contrib/storage-mongo/src/test/java/org/apache/drill/exec/store/mongo/TestTableGenerator.java

```diff
@@ -58,7 +59,12 @@ public static void generateTable(String dbName, String collection,
         .jsonArray(jsonArray).importFile(jsonFile).build();
     MongoImportExecutable importExecutable = MongoImportStarter
         .getDefaultInstance().prepare(mongoImportConfig);
-    importExecutable.start();
+    MongoImportProcess importProcess = importExecutable.start();
+
+    while (importProcess.isProcessRunning()) {
```

Comment:
Please add a comment explaining the change - Import process is running in a different thread/process and we do not to start running any test prior to import complete?